numba/np/arraymath.py

```diff
@@ -3303,6 +3303,18 @@ def np_imag_impl(a):
 #----------------------------------------------------------------------------
 # Misc functions
 
+@overload_method(types.Array, "__contains__")
+def np_contains(arr, key):
+
+    if not isinstance(arr, types.Array):
+        return
+
+    def np_contains_impl(arr, key):
+        return key in arr
+
+    return np_contains_impl
```

Comment:
I think the reason all the CI jobs are failing is because there's no logic implemented for dispatching to `__contains__` or to do the actual work for `key in arr`.